RecoLocalCalo/HGCalRecProducers/python/HGCalRecHit_cfi.py

```diff
@@ -79,8 +79,8 @@
 
     # EM Scale calibrations
     layerWeights = dEdX_weights,
-    thicknessCorrection = cms.vdouble(0.964,0.920,0.909), # 100, 200, 300 um
-    
+    thicknessCorrection = cms.vdouble(1.132,1.092,1.084), # 100, 200, 300 um
```

Comment:
Hi all,
indeed @amartelli the documentation that you provided is extremely useful.
While I agree on the general principle of partitioning PR in pure technical and PR that also change the physics content, having had those links 1 week ago would have allowed, IMHO, the integration of this PR as 1 unique block, instead of going back-and-forth as we did.